website/graphql/Connections.md

```diff
@@ -310,29 +310,31 @@ The server must provide a type called `PageInfo`.
 of which return non-null booleans.
 
 `hasPreviousPage` is used to indicate whether more edges exist prior to the set
-defined by the clients arguments. If the client is paginating with `after`, then
-the client may determine if prior edges exist, if it can do so efficiently. If
-the client is paginating with `last`, then the server must determine if prior
-edges exist. Otherwise, the server must return `false`. More formally:
+defined by the clients arguments. If the client is paginating with 
+`last`/`before`, then the server must return {true} if prior edges exist, 
+otherwise {false}. If the client is paginating with `first`/`after`, then the 
+client may return {true} if edges prior to `after` exist, if it can do so 
+efficiently, otherwise may return {false}. More formally:
 
 HasPreviousPage(allEdges, before, after, first, last):
   * If {last} is set:
     * Let {edges} be the result of calling {ApplyCursorsToEdges(allEdges, before, after)}.
-    * If {edges} contains more than {last} elements, return {true}.
+    * If {edges} contains more than {last} elements return {true}, otherwise {false}.
```

Comment:
It's not a big deal either way (it doesn't affect our use case at all) but I'd originally left this to fall through intentionally. In the (admittedly bizarre) case where the client provides only `last` and `after`, and there are fewer than `last` nodes after `after`, I figured it made sense for the server to still be able to return `true` if it can efficiently determine that there are elements prior to `after`.